openml/runs/trace.py

```diff
@@ -448,3 +464,14 @@ def __str__(self):
             self.evaluation,
             self.selected,
         )
+
+    def __eq__(self, other):
+        if not isinstance(other, OpenMLTraceIteration):
+            return False
+        attributes = [
+            'repeat', 'fold', 'iteration', 'setup_string', 'evaluation', 'selected', 'paramaters',
```

Comment:
I noticed the typo `paramaters` here, and also noticed this is present all throughout the init method. Could you fix that up there too?
Also, it looks like this code probably isn't tested then? Since `getattr` should raise an exception as this object does not have an attribute `paramater` as (somehow) the attribute to the object itself _is_ spelled correctly :)

Reply:
I fixed the attribute and removed the equals function as it was not used.